src/coffee/clients_panel.coffee

```diff
@@ -148,14 +143,8 @@ class ClientsPanel extends Panel
 
   template: Handlebars.compile('
     <div>
-      <div class="top-bar">
-        <span class="podlove-subscribe-back-button">&lsaquo;</span>
-        <img src="{{scriptPath}}/images/icon-big@2x.png">
-        <span class="panel-title">{{t "panels.title"}}</span>
-      </div>
       <div class="device-cloud-switch">
-        <button class="podlove-subscribe-local active">{{t "clients_panel.app"}}</button>
-        <button class="podlove-subscribe-cloud">{{t "clients_panel.cloud"}}</button>
+        <button class="podlove-subscribe-local active">{{t "clients_panel.app"}}<span class="podlove-subscribe-tab-active"></span></button><button class="podlove-subscribe-cloud">{{t "clients_panel.cloud"}}<span class="podlove-subscribe-tab-active"></span></button>
```

Comment:
Okay, sure. I had to add comments instead, to make the two `display: inline-block` and `width: 50%` buttons fit into one line.